rootfs/mkrootfs_debian.sh

```diff
@@ -15,6 +17,45 @@ if [ "$(id -u)" != 0 ]; then
 	exit 1
 fi
 
+function usage() {
+    echo "Usage: $0 [-a | --arch architecture] [-h | --help]
+
+Build a Debian chroot filesystem image for testing libbbpf in a virtual machine.
+By default build an image for the architecture of the host running the script.
```

Comment:
It seems contradictory to say that the default arch is that of the host and then specifying that it's `amd64`.

Reply:
ha, good point. I originally defaulted to `uname -m`, unfortunately, Debian uses amd64, arm64... instead of x86_64, aarch64 so I ended up defaulting to amd64.

Anywhay... that got me to dig a bit more as to how debootstrap would figure out which architecture to use in Debian lingua... this got me to find
```
$ /usr/bin/dpkg --print-architecture
amd64
```